infogami/utils/storage.py

```diff
@@ -206,25 +206,39 @@ class DictPile(DictMixin):
     """
     def __init__(self, dicts=[]):
         self.dicts = dicts[:]
-        
+
+    def __len__(self):
+        return len(self.dicts)
+
+    def __iter__(self):
+        for i in self.dicts:
```

Comment:
This should be an iterator over the merged dicts, but I'm not sure it's worth the aggravation to implement.